docs/local_development_monitoring.adoc

```diff
@@ -40,286 +39,67 @@ sudo systemctl start docker
+-rwxr-xr-x. 1 user1 user1  96989318 Dec 13 16:50 /home/user1/bin/oc-1.3.2
+-rwxr-xr-x. 1 user1 user1 107440536 Feb 13 15:36 /home/user1/bin/oc-1.4.1
 ----
 
-*Note: the container will stay in the foreground (on purpose). This allows you to see the container startup messages and allows you to exit the container by pressing ctrl-c*
-
-== Setting up the ZAIO
-The ZAIO is a completely default installation of zabbix. It has the default templates, items and triggers loaded. The reason it's completely default is that the ZAIO is used by multiple teams, for multiple projects. Therefore, having a default install makes sense.
-
-The ZAIO is also a completely ephemeral. Meaning that when the ZAIO container is stopped and then re-started, it is back to being a completely default installation of zabbix. This is great for development.
+== Containers
+=== Locating the containers
+The local development environment uses CentOS versions of the containers found in https://github.com/openshift/openshift-tools/tree/prod/docker. These containers are periodically built and pushed to the public Docker hub at https://hub.docker.com/u/openshifttools/ (the local development setup scripts will automatically download them).
 
-This makes the ZAIO a great resource for local development.
+If you are interested in modifying the containers yourself, you can build them from the Dockerfiles in https://github.com/openshift/openshift-tools/tree/prod/docker (just be sure to tag your locally built containers with the same names that the `start-local-dev-env.sh` and the `zabbix_monitoring_cent7_local_dev.yaml` template tries to pull/reference. See https://github.com/openshift/openshift-tools/tree/prod/docker/README.adoc for more details.
 
-=== Setting up the openshift-tools Ansible Zabbix Modules
-In order to setup the openshift-tools Ansible Zabbix modules, you need to either install the Zabbix Ansible Module package, or setup the python path to point to the openshift_tools python module.
+=== Running the monitoring containers locally
+Go into openshift-tools/docker/local_development and run the `start-local-dev-env.sh` script.
 
-For development purposes, it's almost always preferable to set the python path, as then you're guaranteed to have the latest code.
-
-.Steps to configure the Ansible Zabbix Modules:
-. Change into the openshift-tools git repo
-[source]
-----
-> cd openshift-tools/ansible/playbooks/adhoc/zabbix_setup
-----
-. Set the python path to see our openshift_tools python package, as well as the system's site-packages
-[source]
 ----
-> export PYTHONPATH=$(pwd)/../../../..:$(ls /usr/lib/python2*/site-packages/ -d)
+cd openshift-tools/docker/local_development
+./start-local-dev-env.sh
 ----
 
+This will do the following:
+* Open a firewall port on port 53 so that OpenShift containers can do DNS resolution to other OpenShift containers. The change is ephemeral, so restarting firewalld should undo the change.
```

Comment:
These are not seen as bullit points in the rendered doc here:
https://github.com/joelddiaz/openshift-tools/blob/3212168bde24602fd92d9e9f94a1a81872e74682/docs/local_development_monitoring.adoc#running-the-monitoring-containers-locally